src/reader.rs

```diff
@@ -155,11 +155,11 @@ impl ParquetFile {
             .unwrap_or_else(|| Err(Error::from(self.path.to_path_buf())))
     }
 
-    pub fn metadata(&self) -> Result<ParquetMetaDataPtr> {
+    pub fn schema(&self) -> Result<Type> {
```

Comment:
I believe there are other places where this func is being used.

if you run the following command you should see where it fails: 
```
cargo clean
cargo test
```

Reply:
The only other places this is used is in the tests - if it's not used in the application code anymore are you happy for me to just fix up the test?

The reason for the change is that it's no longer possible (that I can find..) to acquire an owned [`parquet::file::metadata::ParquetMetaData`](https://docs.rs/parquet/1.0.0/parquet/file/metadata/struct.ParquetMetaData.html), only a reference.